src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.c

```diff
@@ -9,7 +9,33 @@ void EnBji01_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnBji01_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnBji01_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+/* Init helper functions */
+void func_809CCE98(EnBji01* this, GlobalContext* globalCtx); /* Level 1 */
+void func_809CCEE8(EnBji01* this, GlobalContext* globalCtx); /* Level 2 */
+void func_809CD028(EnBji01* this, GlobalContext* globalCtx); /* Level 3 */
+void func_809CD328(EnBji01* this, GlobalContext* globalCtx); /* Level 4 */
+void func_809CCDE0(EnBji01* this, GlobalContext* globalCtx); /* Level 5 */
+void func_809CD634(EnBji01* this, GlobalContext* globalCtx); /* Level 5 */
+void EnBji01_DoNothing(EnBji01* this, GlobalContext* globalCtx); /* Level 6 */
+void func_809CD6C0(EnBji01* this, GlobalContext* globalCtx); /* Level 3 */
+void func_809CD70C(EnBji01* this, GlobalContext* globalCtx); /* Level 4 */
+void func_809CD77C(EnBji01* this, GlobalContext* globalCtx); /* Level 5 */
+
+/* Draw helper functions */
+s32 EnBji01_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* actor);
+void EnBji01_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* actor);
+
+/* Segmented data */
```

Comment:
The comments labelling helper functions and data are unneeded

Reply:
Will be fixed in a batch commit.